cmd/wait-for-github/ci.go

```diff
@@ -42,31 +42,62 @@ type ciConfig struct {
 var (
 	// https://regex101.com/r/dqMmDP/1
 	commitRegexp = regexp.MustCompile(`.*github\.com/(?P<owner>[^/]+)/(?P<repo>[^/]+)/commit/(?P<commit>[abcdef\d]+)/?.*`)
+	prRegexp     = regexp.MustCompile(`.*github\.com/(?P<owner>[^/]+)/(?P<repo>[^/]+)/pull/(?P<prnumber>[\d]+)/?.*`)
 )
 
-type ErrInvalidCommitURL struct {
+type ErrInvalidURL struct {
 	url string
 }
 
-func (e ErrInvalidCommitURL) Error() string {
-	return fmt.Sprintf("invalid commit URL: %s", e.url)
+func (e ErrInvalidURL) Error() string {
+	return fmt.Sprintf("invalid URL to either PR or commit: %s", e.url)
+}
+
+func extractRefFromPrURL(url string) (owner, repo, ref string) {
```

Comment:
I don't think we use named return parameters anywhere else in this project.
 
I do think it makes the type signature clearer, so we can keep that bit, but can we still `return` things explicly?